.coveragerc

```diff
@@ -3,6 +3,8 @@ branch = True
 data_file = .coverage
 source=commerce_coordinator
 omit =
+    */tests/*
```

Comment:
It is not recommended to omit test files.

Reply:
Phil got that